gcovr/utils.py

```diff
@@ -47,7 +47,7 @@ def already_visited(self, path):
     DOS_DEVICE_PATH_PREFIX = "\\\\?\\"
     DOS_DEVICE_PATH_PREFIX_UNC = DOS_DEVICE_PATH_PREFIX + "UNC\\"
 
-    def realpath(path):
+    def _realpath(path):
```

Comment:
Why was this function renamed? A leading underscore typically indicates that the variable would be private, but we do want to import this function e.g. in the HTML writer.

Reply:
Nowhere in the code realpath shall be used anymore except in the filter therefor it's now private to the module. Maybe it's even better to add it as private method to the filter.

Reply:
Ok, you're right with the HTML writer. I'll revert the renaming.